modules/admin_manual/pages/configuration/files/encryption/encryption_configuration_quick_guide.adoc

```diff
@@ -112,6 +115,18 @@ They need to:
 {occ-command-example-prefix} maintenance:singleuser --off
 ----
 
+If you have a large instance with many users, use this to decrypt the files:
+
+* Set the variable as export `OC_RECOVERY_PASSWORD=1111`, then run this set of commands:
```

Comment:
Example on how to export it? Like `export OC_RECOVERY_PASSWORD=1111 `

Reply:
example added